common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -1971,4 +1974,31 @@ object CelebornConf extends Logging {
         s"celeborn.columnar.shuffle.encoding.dictionary.maxFactor)`.")
       .doubleConf
       .createWithDefault(0.3)
+
+  val RPC_CACHE_SIZE: ConfigEntry[Int] =
+    buildConf("celeborn.rpcCache.Size")
+      .categories("client")
+      .withAlternative("rss.rpc.cache.size")
+      .version("0.2.0")
+      .doc("The max cache items count for rpc cache.")
+      .intConf
+      .createWithDefault(256)
+
+  val RPC_CACHE_CONCURRENT_LEVEL: ConfigEntry[Int] =
+    buildConf("celeborn.rpcCache.concurrent.level")
+      .categories("client")
+      .withAlternative("rss.rpc.cache.concurrent.level")
+      .version("0.2.0")
+      .doc("The number of write locks to update rpc cache.")
+      .intConf
+      .createWithDefault(32)
+
+  val RPC_CACHE_EXPIRE: ConfigEntry[Long] =
+    buildConf("celeborn.rpcCache.expire")
```

Comment:
I'm a little worried about the naming, as we already have `celeborn.rpc.maxParallelism`, what do you think

```
celeborn.rpc.cache.size/capacity
celeborn.rpc.cache.concurrentLevel
celeborn.rpc.cache.expireTime
```
?

Reply:
Ref: Spark has configuration key named `spark.rdd.checkpoint.cachePreferredLocsExpireTime`